packages/manager/apps/pci-gateway/src/pages/add/SizeStep.tsx

```diff
@@ -101,32 +102,42 @@ export const SizeStep = (): JSX.Element => {
           <div className="grid grid-cols-1 gap-2 text-left text w-full">
             <div className="">
               <OsdsText
-                level={ODS_THEME_TYPOGRAPHY_LEVEL.body}
-                size={ODS_THEME_TYPOGRAPHY_SIZE._400}
+                level={ODS_THEME_TYPOGRAPHY_LEVEL.heading}
+                size={ODS_THEME_TYPOGRAPHY_SIZE._200}
                 color={ODS_THEME_COLOR_INTENT.text}
               >
                 {item.label}
               </OsdsText>
             </div>
-            <hr className="w-full border-solid border-0 border-b border-b-[#85d9fd]" />
             <div className="text-sm">
               <OsdsText
-                level={ODS_THEME_TYPOGRAPHY_LEVEL.body}
+                level={ODS_THEME_TYPOGRAPHY_LEVEL.caption}
                 size={ODS_THEME_TYPOGRAPHY_SIZE._200}
                 color={ODS_THEME_COLOR_INTENT.text}
               >
                 {item.bandwidthLabel}
               </OsdsText>
             </div>
-            <div className="text-sm mt-4">
+            <hr className="w-full border-solid border-0 border-b border-b-[#85d9fd]" />
```

Comment:
You can't do that with Ods ? : https://ovh.github.io/design-system/v17.2.2/?path=/story/ods-components-layout-divider-quark-specifications--page